### DISCUSSION TITLE
This is my review of AppFlowy (as a person that hates cluttered UI)

### DISCUSSION THREAD

**Discussion by ghost** [AppFlowy-IO/AppFlowy#665] [Ideas]
Hi, this is my review about using AppFlowy.

The font is not clear and the contrast of the text and the background isn't pleasing to the eye maybe it's because of the font's colors.
I would recommend a font size and style option to choose from. But the current font size is superb but I really need to change the font contrast in order for me to use it (by the way I reported it already, I'm using Windows 10)

![font](https://user-images.githubusercontent.com/52787179/180102813-e62e7814-e2ee-42c2-a606-895b851d46e3.PNG)

I need to copy paste an emoji to paste it in the Edit Property, an emoji toolbar or emoji pop-up using a shortcut would be good.
![need emoji toolbar](https://user-images.githubusercontent.com/52787179/180101909-c95f0536-c079-4d76-9e80-7ecab9646300.PNG)

It would be also very cool feature if I could link my page to the table using the links or something else? 

![needs improvement](https://user-images.githubusercontent.com/52787179/180101990-18966fcb-dfce-4e92-b942-e40118744bad.PNG)

Feature Request
- A Markdown Editor 
- Font Style and Size Option
- An outline where you can click it and you'll be move to the section of that page (for future feature, but not really necessary I think?)

![outline](https://user-images.githubusercontent.com/52787179/180102590-d8bd013e-7060-4c46-ace0-76130036f61d.PNG)

- Change the Heading Font Style or Size (for future feature, not really necessary right now)

Thoughts and Comments

I think this would be good alternative for Notion, Evernote and Obsidian and I think it has a very great potential to beat the Notion as a bulky, cluttered and complicated functionality and UI. 

The UI of AppFlowy is very clean and uncluttered. I really like it and it's better than Obsidian, Evernote, Notion, Craft Docs - this is only my opinion (and don't hate me please) because I hate clean and cluttered note taking app environment because it's not good for my mental health. Anyway, I love this app and I'm looking forward to it.

**Comment by annieappflowy** [comment]
Very much appreciate @wharjack your review with a lot of details. Here I'm pulling out feature requests from this review:

If it's planned, I will put a link next to the request down below:
- A Markdown Editor: https://github.com/orgs/AppFlowy-IO/projects/5/views/5
- Font Style and Size: https://github.com/AppFlowy-IO/AppFlowy/issues/653

Other feature requests that need to assess:
- outline for a page
- link to a page

**Comment by Drallas** [comment]
+1 for the Font Style and Size Option

**Comment by annieappflowy** [comment]
@Drallas , thanks for the feedback. We now allow the user to choose the font size from small, medium, and large. If you can give it a try and let us know your thoughts, that would be super helpful for us!
<img width="275" alt="image" src="https://user-images.githubusercontent.com/12026239/210123470-da38f25e-aef1-4593-9dc0-bcf53995e274.png">

**Reply by Drallas** [reply]
Thanks, that works, but only on the content page (right pane); but the navigation (left pane) stays the same font. 

It Would be nice if 'command + / -' could be used to increase the font-size of the GUI.

**Comment by annieappflowy** [comment]
@wharjack , there was a major upgrade to the editing experience. https://github.com/AppFlowy-IO/AppFlowy/releases
Let us know your thoughts!

**Reply by ghost** [reply]
Hi, I'm going to use and test it, and I'll update you guys for my feedback. The large sized font option is excellent. I would like to suggest a function like "Ctrl" + "+" then the UI would zoom in including all text, just like obsidian. Thank you!

**Reply by annieappflowy** [reply]
Thanks @wharjack @Drallas 
Your feedback are valuable and important to us. We will iterate on the font size based on your input.
